script2.js

```diff
@@ -0,0 +1,14 @@
+/**
```

Comment:
Maybe try to get in the habit of naming the files something more descriptive.
Naming things is a BIG part of development and its a skill that you will want to hone.
Its easy to get stuck scratching your head on a logical and simple names so any practice is good practice.